deployment/common.build.sh

```diff
@@ -36,9 +36,9 @@ build_jellyfin()
 
     echo -e "${CYAN}Building jellyfin in '${ROOT}' for ${DOTNETRUNTIME} with configuration ${CONFIG} and output directory '${OUTPUT_DIR}'.${NC}"
     if [[ $DOTNETRUNTIME == 'framework' ]]; then
-        dotnet publish "${ROOT}" --configuration "${CONFIG}" --output="${OUTPUT_DIR}"
+        dotnet publish "${ROOT}" --configuration "${CONFIG}" --output="${OUTPUT_DIR}" "-p:GenerateDocumentationFile=false;DebugSymbols=false;DebugType=none"
```

Comment:
Can we share this option? I thought `common.build.sh` was sourced by all other build scripts, so why copy-paste it all over?

Reply:
The `common.build.sh` script is used in almost all the `build.sh` scripts except the Docker images and the Fedora/Debian packages since they already manage building Jellyfin themselves so i just appended the option for all the Package builds.

Reply:
@JustAMan i changed the Dockerfiles to use the `build_jellyfin`  function.